wb/main/utils/safe_runner.py

```diff
@@ -25,28 +25,36 @@
 from wb.main.jobs.feed.feed_socket_service import FeedSocketService
 
 
+def safe_run_configurable(emit_web_socket: bool):
```

Comment:
Why not to use the same decorator name with default `emit_on_error` argument set tot true?

Reply:
fixed